src/plugins/plugin.legend.js

```diff
@@ -72,6 +72,12 @@ defaults._set('legend', {
 				};
 			}, this);
 		}
+	},
+
+	title: {
+		display: false,
+		position: 'center',
+		value: '',
```

Comment:
Would `text` be better name, if we try to make these consistent?

Reply:
Sure, I can go with that